cardinal/message/manager.go

```diff
@@ -75,11 +76,11 @@ func (m *Manager) GetMessageByType(mType reflect.Type) (types.Message, bool) {
 	return msg, ok
 }
 
-// isMessageNameUnique checks if the message name already exist in messages map.
-func (m *Manager) isMessageNameUnique(msgName string) error {
```

Comment:
Then, this would be `isMessageIDUnique`

Reply:
i put the group / name validation in NewMessageType. i want the error to happen right away if they construct a message with a bad group and/or name. the isMessageNameUnique is more of a stateful check, whereas group/name validation is a stateless check.